cmd/create/cluster/cmd.go

```diff
@@ -3348,14 +3347,6 @@ func clusterConfigFor(
 	return clusterConfig, nil
 }
 
-func validateBillingAccount(billingAccount string) error {
-	if billingAccount == "" || !ocm.IsValidAWSAccount(billingAccount) {
-		return fmt.Errorf("billing account is invalid. Run the command again with a valid billing account. %s",
```

Comment:
(assuming it's the acct _number_ that's not valid,) consider changing to: Billing account number is not valid. Rerun the command with a valid billing account number.

Also, good error messages provide user help if the retry doesn't work. what can the user do if they still get an error?

Reply:
Hi @jseseCCS , I've updated the error message with the one you suggested.

We are planning to add a `rosa list billing-accounts` command (tracked via OCM-9642).

I was thinking we could revisit this error message after we implemented that.

The final result would be something like:

```
Billing account number is not valid. Rerun the command with a valid billing account number. 
To see the list of billing account options, run the rosa list billing-accounts command.
```

Thoughts?

Reply:
Sounds good!